logic/src/commonMain/kotlin/com/wire/kalium/logic/configuration/appVersioning/AppVersionRepository.kt

```diff
@@ -17,7 +17,7 @@ internal class AppVersionRepositoryImpl(
         wrapApiRequest { api.fetchAppVersionBlackList(blackListUrl) }
             .fold({
                 kaliumLogger.e("$TAG: error while fetching VersionBlacklist: $it")
-                true
+                false
```

Comment:
so if there's an error, for instance no network, then if we return false, we allow the user to use the version that can be blacklisted but we don't yet know about it - is that ok?